stylesheets/partials/page-specific/citysourced/citysourced-reporter.scss

```diff
@@ -286,4 +287,15 @@
             }
         }
     }
+}
+
+@media screen and (max-width: $screen-xs) {
+	.bc_alert {
+		p {
+			font-size: .9em;
+		}
+		i {
+			font-size: 1.33em;
```

Comment:
Should there be a $fa-lg variable for this?

Reply:
we aren't including any variables for font-awesome at this point. if we start doing more responsive icons, than i think that would be a good idea.